### ISSUE TITLE
Bug: Disable not working with input only with textarea

### ISSUE THREAD

**Issue by norder00** [sbb-design-systems/sbb-angular#193]
**Adding disabled in the input field is not working**

In the *input* tag of *sbb-field* when we add **disabled** it will not disable the field. 
When we do the same for *textarea* it will be disabled. 

Example schreenshot: 
![image](https://user-images.githubusercontent.com/52195282/66301268-147fd980-e8f7-11e9-8c1d-63a844fb574e.png)

You can test it on the sbb angular example too: https://sbb-angular.app.sbb.ch/latest/public/components/field

If you click on the Disablet checkbox, no field will be disabled. 

Here is our example where we used it, you can checkout the branch: https://code.sbb.ch/projects/KD_PSDR/repos/psdr-frontend/commits/3dcc70d0fa2e967b9e2425224ea741b5b9f98208#src/app/modules/tab/components/formbuilder/formbuilder.component.html

inputs are not disabled only the textarea.

I would very much appreciate if the bug would be fixed as soon as possible, thank you.

**Comment by kyubisation** [comment]
Hello @norederguti 
Please use the official bug template when creating a bug issue: https://github.com/SchweizerischeBundesbahnen/sbb-angular/issues/new?assignees=&labels=bug&template=bug_report.md&title=Bug%3A+

Disabling an input field seems to work fine: https://stackblitz.com/edit/angular-sbb-ebc2sv?file=src%2Fapp%2Fapp.component.html
Can you provide a stackblitz reproduction?

However I will investigate why the example does not work.

**Comment by jeripeierSBB** [comment]
Showcase example fixed with #333 
Disabling input fields in reactive forms generates following console warning, so use mentioned approach:
```     It looks like you're using the disabled attribute with a reactive form directive. If you set disabled to true
      when you set up this control in your component class, the disabled attribute will actually be set in the DOM for
      you. We recommend using this approach to avoid 'changed after checked' errors.
      
      Example:
      form = new FormGroup({
        first: new FormControl({value: 'Nancy', disabled: true}, Validators.required),
        last: new FormControl('Drew', Validators.required)
      });```
